### DISCUSSION TITLE
Bug - can’t open any repos or view pull requests

### DISCUSSION THREAD

**Discussion by rbaturov** [community/community#146775] [Mobile]
There is this bug in the mobile app, when a fix is expected to land?

**Comment by MindfulLearner** [comment]
Is it the first time youre using app? 

i mean the app for me was strangely harder than normal haha. But normally the app is okay still giving you the issue?

**Reply by rbaturov** [reply]
Not the first time, occurred after an update a week ago
Still experiencing this issue

**Reply by MindfulLearner** [reply]
Mhm report the issue.

**Reply by rbaturov** [reply]
Where?

**Reply by MindfulLearner** [reply]
https://support.github.com/ try there!
